indexer_util/indexer_util/indexer_util.py

```diff
@@ -116,6 +116,7 @@ def es_actions(scripts_by_id):
                 # use any string here.
                 '_type': 'type',
                 '_id': _id,
+                'scripted_upsert': True,
```

Comment:
didn't quite get this right in the last PR, we need both `scripted_upsert = True` as well as a blank `upsert` doc.